zulip/integrations/slack/slackdata2zulipdata.py

```diff
@@ -339,21 +339,38 @@ def main(slack_zip_file: str) -> None:
     zerver_message = []
     zerver_usermessage = []
 
+    # check if a user has been mentioned in a message
+    def check_user_mention(text):
+        mentions = re.findall(r'(@(?:\*\*([^\*]+)\*\*|(\w+)))', text)
```

Comment:
The regex should be the same with the one at L230, shouldn't it?

Reply:
Here, we are taking the text from zerver_message which already has the sanitized text. So it'll be in the form `@**full_name**`. This is the regex for detecting this.

Reply:
IC